src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -405,6 +420,8 @@ private async Task StopAsyncCore(bool disposing)
                     (_serviceProvider as IDisposable)?.Dispose();
                     _disposed = true;
                 }
+
+                _connectionId = null;
```

Comment:
Not sure if we care about this, lets see what other people think

Reply:
I like nulling it out once the connection is stopped.

Reply:
Stopped or disposed?

Reply:
Stopped. The ConnectionId isn't that useful after you've disconnected, and it can take on multiple values prior to being disposed.